src/IconToggle/IconToggle.react.js

```diff
@@ -33,7 +33,7 @@ const propTypes = {
     /**
     * Name of icon to show
     */
-    name: PropTypes.string.isRequired,
+    name: PropTypes.string,
```

Comment:
The commit made was fixing the ci. It did fix the ci, but it breaks the <ListItem> component. This PR I made fixes `<ListItem>` and the CI also works